.travis.yml

```diff
@@ -5,25 +5,23 @@ language: generic
 
 matrix:
   include:
-    - python: "2.7"
-      env: PYTAVE=no  SYMPY_VER=1.0 OCT_PPA=yes DOCTEST=yes COLUMNS=80
-    - python: "3.5"
-      env: PYTAVE=no  SYMPY_VER=1.0 OCT_PPA=yes DOCTEST=yes COLUMNS=80
+    - env: PYTHON=python2 PYTHON_VER="" PYTAVE=no  SYMPY_VER=1.0 OCT_PPA=yes DOCTEST=yes COLUMNS=80
+    - env: PYTHON=python3 PYTHON_VER="3" PYTAVE=no  SYMPY_VER=1.0 OCT_PPA=yes DOCTEST=yes COLUMNS=80
 
 # need octave devel pkgs for doctest (has compiled code as of July 2015)
-install:
+before_install:
   - if [ "x$OCT_PPA" = "xyes" ]; then
         sudo apt-add-repository -y ppa:octave/stable;
     fi
-  - sudo apt-get update -qq -y;
-  - sudo apt-get install -qq -y octave liboctave-dev;
```

Comment:
Apparently it was hiding useful information. Please note that travis will collapse those commands in the web view, so it should not hurt too much?